packages/editor/src/client/api/subscription.graphql

```diff
@@ -1,77 +1,92 @@
 fragment FullSubscription on Subscription {
-    id
-    createdAt
-    modifiedAt
+  id
```

Comment:
What is the story with formatting? Are you using other formatting then we had? We should all use the same agreed so we don't go back and forth with it.